views/networking/firewall/networking.erb

```diff
@@ -69,6 +71,15 @@ edit_perm = has_permission?("Firewall:edit", @firewall) %>
                 }
               ) %>
             </td>
+            <td class="whitespace-nowrap py-4 pl-4 pr-3 text-sm font-medium text-gray-900 sm:pl-6" scope="row">
```

Comment:
We should make these firewall rules editable, like postgres firewall rules. `views/postgres/networking.erb` has examples on how to do it.

It is useful for modifying firewall descriptions (and IP ranges) without removing/re-adding the firewall rules.